src/lib/layer.js

```diff
@@ -116,37 +116,29 @@ export default class Layer {
     }
   }
 
-  // If state has a model, draw it with supplied uniforms
-  /* eslint-disable max-statements */
-  pick({info, uniforms, pickEnableUniforms, pickDisableUniforms, mode}) {
-    const {gl} = this.context;
-    const {model} = this.state;
+  // called on mouse event.
+  // @returns {bool} handled
+  pick({info, mode}) {
```

Comment:
Consider change the name of this function to "processPickingInfo". This function on longer "picks" any more - the pick action itself is happening in "drawLayers" now.